README.md

```diff
@@ -39,6 +39,19 @@ linker = "/Users/yourusername/opt/bin/x86_64-pc-elf-gcc"
 
 Where `yourusername` is your username.
 
+This project uses the xargo crate. To get it:
+
+ Before install necessary dependencies, for Ubuntu 16.04, we need to install libssl-dev:
```

Comment:
This is probably better as a note, something like `xargo has a few dependencies, check its readme to make sure you've got them installed.